File: swapp.cpp
#include<iostream>
using namespace std;


void alter(int arr[], int n)

{
    int start = 0;
    while(start < n-1)
    {
        swap(arr[start], arr[start +1]);
        start += 2;

    }
}

void printArray(int arr[], int n )

{
    for (int i = 0; i<n ; i++)

    {
        cout<<arr[i]<<" ";

    }

    cout<<endl;
}

    int main()
    {
        int arr[6] = {2 , 4, 5 , 6 , 9 ,3};
        alter(arr , 6);

        printArray(arr,6);


return 0 ;

    }

File: swapp.java
import java.util.*;

public class tempCodeRunnerFile{
    static void swapalter(int[]arr ,int size)
    {
        for(int i = 0;i<size;i += 2)
        {
            if(i+1 <size)
            {
                int temp = arr[i];
                arr[i] = arr[i+1];
                arr[i+1] = temp;
            }
        }
    }

    static void printArray(int[] arr)
    {
        for(int num :arr)
        {
            System.out.print(num + " ");

        }
        System.out.println();

    }


    public static void main(String[]args)
    {
        int[] odd = {2,3,4,5,9};
        int[] even = {2,3,4,9};


        swapalter(odd,odd.length);
        printArray(odd);


        swapalter(even,even.length);
        printArray(even);
    }
}
